### PR TITLE
Buttons+card

### PR DESCRIPTION
**Pull request by vp5h**
https://avidcss.netlify.app/components/buttons/buttons
https://avidcss.netlify.app/components/cards/card
- Documentation
- All the Cards
- Buttons
- Utility Classes

### CODE REVIEW

**Review by Sudhanshusoni45 — APPROVED** [review]

File: Components/Cards/Cards.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  border-radius: 1rem;
+  flex-direction: column;
+  box-shadow: 0 2px 1px -1px rgb(0 0 0 / 20%), 0 1px 1px 0 rgb(0 0 0 / 14%),
+    0 1px 3px 0 rgb(0 0 0 / 12%);
+}
+
+.card_horizontal {
+  position: relative;
+  color: inherit;
+  background-color: #ffffff;
```
Comment (Sudhanshusoni45) [inline Components/Cards/Cards.css:18]: Suggestion : use color variable instead of hard code

File: Components/Buttons/Buttons.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  letter-spacing: 0.5px;
+  background: grey;
+  font-family: "roboto";
+  font-size: 1rem;
+  height: 3rem;
+  box-shadow: rgb(0 0 0 / 17%) 0px 4px 9px;
+}
+
+.icon {
+  color: var(--primary-color) !important;
```
Comment (NagaSatyaGeeth) [inline Components/Buttons/Buttons.css:15]: i remember Vikram suggesting to not use `!important` so do check this once

File: Components/Buttons/buttons.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                <!-- <li><a href="../Colors/colors.html">Colors</a></li>
+          <li><a href="../Typography/typography.html">Typography</a></li> -->
+            </ul>
+
+            <div class="list-topic">
+                <h4 class="highlight">Components</h4>
+            </div>
+            <ul>
+                <li><a href="../Cards/card.html">Card</a></li>
+                <!-- <li><a href="./Components/Avatar/avatar.html">Avatar</a></li>
```
Comment (NagaSatyaGeeth) [inline Components/Buttons/buttons.html:50]: another suggestion is to remove commented code

File: Components/Buttons/Buttons.css (hunk 2 above)
Comment (vp5h) [reply Components/Buttons/Buttons.css:15]: I have enforced a diffrent font color all across and it was interfering with component font color

File: Components/Buttons/buttons.html (hunk 3 above)
Comment (vp5h) [reply Components/Buttons/buttons.html:50]: noted. Will uncomment to make routes for more components

File: Components/Cards/Cards.css (hunk 1 above)
Comment (vp5h) [reply Components/Cards/Cards.css:18]: Sure

File: Components/Buttons/Buttons.css (hunk 2 above)
Comment (NagaSatyaGeeth) [reply Components/Buttons/Buttons.css:15]: yeah got it

**Review by NagaSatyaGeeth — APPROVED** [review]

**Pull request merged**